src/main/java/lotto/controller/LottoMain.java

```diff
@@ -1,41 +1,115 @@
     }
 
     private static UserPurchase getUserPurchaseInput() {
         try {
-            return InputView.getUserPurchase();
+            int purchasePrice = InputView.getPurchasePrice();
+            int manualTicketCount = getManualTicketCountInput();
+            return new UserPurchase(purchasePrice, manualTicketCount);
         } catch (Exception e) {
             System.out.println(e.getMessage());
             return getUserPurchaseInput();
         }
     }
 
+    private static List<LottoTicket> getManualLottoTicketInput(UserPurchase userPurchase) {
+        List<LottoTicket> manualLottoTickets = new ArrayList<>();
+        InputPrinter.printManualTicketInputGuideMessage();
+        try {
+            addLottoTicketInput(userPurchase, manualLottoTickets);
+            return manualLottoTickets;
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            return getManualLottoTicketInput(userPurchase);
+        }
+    }
+
+    private static void addLottoTicketInput(UserPurchase userPurchase, List<LottoTicket> manualLottoTickets) {
+        for (int i = 0; i < userPurchase.getManualTicketCount(); i++) {
+            manualLottoTickets.add(getLottoTicketInput());
+        }
+    }
+
+    private static LottoTickets generateManualTicket(LottoGenerator lottoGenerator, List<LottoTicket> manualLottoTickets) {
+        lottoGenerator.setGenerateStrategy(new ManualStrategy(manualLottoTickets));
+        return lottoGenerator.generateTickets();
+    }
+
+    private static LottoTickets generateAutoTicket(LottoGenerator lottoGenerator, UserPurchase userPurchase) {
+        lottoGenerator.setGenerateStrategy(new AutoStrategy(userPurchase.getAutoTicketCount()));
+        return lottoGenerator.generateTickets();
+    }
+
+    private static int getManualTicketCountInput() {
+        try {
+            return InputView.getManualTicketCount();
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            return getManualTicketCountInput();
+        }
+    }
+
     private static WinningLottoNumbers getWinningLottoNumbersInput() {
+        LottoTicket winningLottoTicket = getWinningLottoTicketInput();
+        LottoNumber bonusNumber = getBonusNumberInput();
+        return new WinningLottoNumbers(winningLottoTicket, bonusNumber);
+    }
+
+    private static LottoTicket getWinningLottoTicketInput() {
+        try {
+            InputPrinter.printWinnerLottoNumbersInputGuideMessage();
+            return getLottoTicketInput();
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            return getWinningLottoTicketInput();
+        }
+    }
+
+    private static LottoTicket getLottoTicketInput() {
+        List<Integer> numbers = InputView.getLottoNumbers();
+        List<LottoNumber> lottoNumbers = numbers.stream().map(LottoNumber::new).collect(Collectors.toList());
```

Comment:
```suggestion
        List<LottoNumber> lottoNumbers = numbers.stream()
                .map(LottoNumber::new)
                .collect(Collectors.toList());
```

stream은 가독성을 위해 한줄에 하나씩 쓰는게 좋아요 :)